paasta_tools/utils.py

```diff
@@ -1132,6 +1132,12 @@ def get_docker_client():
         return Client(base_url=get_docker_host(), **client_opts)
 
 
+def get_running_mesos_docker_containers():
+    client = get_docker_client()
```

Comment:
Why did you make this not a parameter when you moved this method from kill_orphaned_docker_containers.py?